bin/alluxio-mount.sh

```diff
@@ -112,7 +112,15 @@ function mount_ramfs_linux() {
       exit 1
     fi
   else
-    mkdir -p ${TIER_PATH}
+    if [[ "$1" == "SudoMount" ]]; then
+      sudo mkdir -p ${TIER_PATH}
+    else
```

Comment:
if "Mount", nothing is really executed in sudo mode.
Actually I don't really know when we use "Mount".
I simply leave this unchanged to match our contract before.